CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ option(run_unittests "set run_unittests to ON to run unittests (default is OFF)"
 option(skip_samples "set skip_samples to ON to skip building samples (default is OFF)[if possible, they are always built]" OFF)
```

Comment:
One more to remove:

jenkins\windows_c.cmd:13:call build.cmd --use-websockets %*